tools/analyzer_plugin/tool/travis.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# ##################################################################################
+#
+# Runs all analyzer plugin tasks that should be run in the parent over_react CI
+#
+# ##################################################################################
+
+# Fail any script if any unexpected errors occur in the subsequent commands
+set -e
+
+pub get
+pub run dart_dev format --check
+pub run dart_dev analyze
+pub run dependency_validator --no-fatal-pins
```

Comment:
FYI @joebingham-wk once this gets merged into you branch, it needs to be updated to run unit tests